WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/CommentUserNoteBlock.java

```diff
@@ -76,12 +78,21 @@ public View configureView(View view) {
             String imageUrl = GravatarUtils.fixGravatarUrl(getNoteMediaItem().optString("url", ""), getAvatarSize());
             noteBlockHolder.avatarImageView.setImageUrl(imageUrl, WPNetworkImageView.ImageType.AVATAR);
             if (!TextUtils.isEmpty(getUserUrl())) {
+                noteBlockHolder.avatarImageView.setOnClickListener(new View.OnClickListener() {
```

Comment:
Shouldn't we set the onClickListener to null in both `else` branches, as the view can be recycled? (same applies to HeaderNoteBlock.java and UserNoteBlock.java)